system-addon/lib/ActivityStream.jsm

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+Components.utils.import("resource:///modules/NewTabURL.jsm");
+
+class ActivityStream {
+  constructor(options) {
+    this.initialized = false;
+    this.options = options;
+  }
+  init() {
+    NewTabURL.override(this.options.newTabURL);
+    this.initialized = true;
+  }
+  uninit() {
+
+    NewTabURL.reset();
```

Comment:
Perhaps the blank line here actually wants to be above uninit()?  Since this is here at all, I kinda suspect eslint isn't up and working for this directory yet, though I don't think we need to fix that here.